demos/browser/webpack.config.hot.js

```diff
@@ -0,0 +1,107 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * This file is a fork of webpack.config.js. 
+ * My apologies.
```

Comment:
We should probably note to make changes in both files when applicable here as well.

Reply:
Got another run, so done.